test_util/errors.py

```diff
@@ -0,0 +1,91 @@
+import abc
```

Comment:
Agreed. I just copied the recordio file from dcos-cli and didn't bother to strip out the unnecessary errors.py dependency. It's gone now.